### DISCUSSION TITLE
JSON broken using Monika config generator

### DISCUSSION THREAD

**Discussion by rgraham7503** [hyperjumptech/monika#782] [General]
**This is a BUG** This used to work, I would paste in my json body and it would create the config and upon running the config it generated it would work. Now when I run the config generated online by Monika config generator I get this error...I have tried to multiple formats of this json and cant get it to post correctly. **This did work previous to latest update.**

`WARN: 2022-07-19T12:12:22.438Z 1 id:9fe991c6-ff2a-4e92-b0b5-dd6f486d161b - POST https://xxxx.com/v2.0/Users -ms, ERROR: You must pass a string or Handlebar
s AST to Handlebars.compile. You passed urn:ietf:params:scim:schemas:core:2.0:User,urn:ietf:params:scim:schemas:extension:xxx:2.0:User,urn:ietf:params:scim:schemas:exten
sion:enterprise:2.0:User,urn:ietf:params:scim:schemas:extension:xxx:2.0:Notification`

![image](https://user-images.githubusercontent.com/46932722/179763021-43990e15-48e4-44eb-9fcf-d80a3f51640f.png)

**Comment by rgraham7503** [comment]
I did get this working manually by **changing Postman** to show code as **nodejs axio** then using that format as the json. I think the config generator still needs fixed as if I paste a normal curl json object it does not create a config file that will work. 
attaching screenshot of actual working json - **PLEASE NOTE the difference in screenshots of config generator and a manually created working json**
![image](https://user-images.githubusercontent.com/46932722/179788824-ed8635c8-580c-461d-b709-7614afc89c7b.png)

**Reply by sapiderman** [reply]
Thanks @rgraham7503, created a ticket. Will get someone to look into it.

**Reply by sapiderman** [reply]
@rgraham7503 Hi, Just FYI. Please take a look at the newest release, v1.10.0. The JSON issue should have been fixed.
Let us know if you find any issues.
